client/styles/_var.scss

```diff
@@ -0,0 +1,23 @@
+// Code inside `@if $is-oldie {}` will compile inside oldie.css
+$is-oldie: false !default;
+
+// Same thing but with mq()
+$mq-breakpoints: (
```

Comment:
FYI, o-grid uses sass-mq so you could probably get rid of these breakpoints to avoid duplication.